crates/bifrost/src/providers/replicated_loglet/remote_sequencer.rs

```diff
@@ -202,11 +204,14 @@ where
             return Ok(current.clone());
         }
 
+        debug!("Renewing connection to {}", current.inner.peer());
         let connection = self
             .networking
             .node_connection(self.params.sequencer)
             .await?;
 
+        Span::current().record("renewed", true);
```

Comment:
I'm not sure this really adds value. Let's remove it. The log message should be sufficient.